### ISSUE TITLE
DAI - search by type of signs doesn't work

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/ASL-DAI#22]
None of the restrictions by type of sign is working.

<img width="674" alt="screen shot 2017-10-06 at 5 59 14 pm" src="https://user-images.githubusercontent.com/13629122/31300156-288cd418-aac0-11e7-958f-9e92d7faaa6e.png">

Gestures seem to be properly excluded when All Signs [excl. Gestures & CL is selected.

But otherwise the selections seem not to restrict the search.

Include compounds (only) works.
Include non-compounds (only) works.

Checking  both should show the full set of signs (the union of the above), but it shows nothing.

Selecting neither include compounds nor include non-compounds should show nothing, but instead shows everything.
<img width="710" alt="screen shot 2017-10-06 at 6 08 25 pm" src="https://user-images.githubusercontent.com/13629122/31300371-603d332a-aac1-11e7-8bf8-d5ac54ce2577.png">

With only Lexical signs selected, all kinds of signs are displayed.
Same for selecting only other types of signs.

Basically, this selection is just completely broken.

**Comment by kwasiopoku** [comment]
Fixed

**Comment by cneidle** [comment]
A search for lexical signs includes fingerspelled signs in the results.

![screen shot 2017-10-16 at 12 58 53 pm](https://user-images.githubusercontent.com/13629122/31624831-fa0788cc-b271-11e7-8a09-4fd60f413f67.png)

![screen shot 2017-10-16 at 12 58 44 pm](https://user-images.githubusercontent.com/13629122/31624836-ff60de36-b271-11e7-8b79-404222fb184b.png)

I checked the first two, and both are  correctly categorized as fingerspelled in the annotations.

**Comment by cneidle** [comment]
WAIT A MINUTE - I just did this search again, and fs- signs are NOT showing up now !!
Please ignore the last entry.  weird.

This does appear to be fixed.  Sorry about that!
